notebooks/COS/LSF/LSF.ipynb

```diff
@@ -0,0 +1,1525 @@
+    "\n",
+    "- `numpy` to handle arrays and functions\n",
+    "- `astropy.io fits` and `astropy.table Table` for accessing FITS files\n",
+    "- `astropy.modeling functional_models` to generate synthetic spectral line shapes\n",
+    "- `astropy.convolution convolve` for a convolution of two discretized functions\n",
+    "<!-- - glob, os, and shutil for working with system files -->\n",
+    "- `astroquery.mast Mast` and `Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "- `scipy.interpolate interp1d` for interpolating two discretized functions to the same sampling\n",
+    "- `matplotlib.pyplot` for plotting\n",
+    "- `urllib` for downloading files stored at a web address online\n",
+    "- `tarfile` for unzipping some compressed STIS data\n",
+    "- `pathlib Path` for managing system paths\n",
+    "\n",
+    "These python packages are installed standard with the the STScI conda distribution. For more information, see our notebook tutorial on [setting up an environment](https://github.com/spacetelescope/COS-Notebooks/blob/master/Setup/Setup.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import for: array manipulation\n",
+    "import numpy as np\n",
+    "\n",
+    "# Import for: reading fits files\n",
+    "from astropy.table import Table\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "# Import for: line profile functions\n",
+    "from astropy.modeling import functional_models\n",
+    "\n",
+    "# Import for: convolutions\n",
+    "from astropy.convolution import convolve\n",
+    "\n",
+    "# Import for: downloading the data\n",
+    "from astroquery.mast import Observations\n",
+    "\n",
+    "#Import for: Interpolating a discretized function\n",
+    "from scipy.interpolate import interp1d\n",
+    "\n",
+    "#Import for: Plotting\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "#Import for: downloading COS' LSF files within python\n",
+    "import urllib \n",
+    "\n",
+    "#Import for: unzipping a compressed file of STIS data\n",
+    "import tarfile\n",
+    "\n",
+    "#Import for: working with system paths\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will also define a few directories we will need:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# These will be important directories for the notebook\n",
+    "\n",
+    "datadir = Path('./data')\n",
+    "outputdir = Path('./output/')\n",
+    "plotsdir = Path('./output/plots')\n",
+    "\n",
+    "# Make the directories if they don't exist\n",
+    "datadir.mkdir(exist_ok=True), outputdir.mkdir(exist_ok=True), plotsdir.mkdir(exist_ok=True)"
```

Comment:
Not a big deal either way, but why not all on one line? I'd consider that more "readable"

Reply:
You mean the comment and code? I know it bugs some people when code lines are >72char or whatever their screens are set to so I try to keep shortish